src/aws_hashicorp_packer_reaper/click_argument_types.py

```diff
@@ -18,3 +19,20 @@ def convert(self, value, param, ctx) -> durations.Duration:
             return durations.Duration(value)
         except ValueError as e:
             self.fail(f'Could not parse "{value}" into duration ({e})', param, ctx)
+
+
+class TagParam(click.ParamType):
+    name = "keyvalue"
+
+    def convert(self, value, param, ctx):
+        splits = value.split("=")
```

Comment:
When there are multiple `=` characters in the string than split will return more than two elements. Please use `split("=", 1)` to avoid an incorrect tag value.